pkg/storaged/client.js

```diff
@@ -1139,10 +1139,10 @@
 function stratis2_fetch_manager_properties(proxy) {
     stratis2_fetch_properties(proxy, ["LockedPoolsWithDevs"]).then(values => {
         if (values.LockedPoolsWithDevs) {
-            proxy.LockedPools = { };
+            proxy.StoppedPools = { };
             for (const uuid in values.LockedPoolsWithDevs) {
                 const l = values.LockedPoolsWithDevs[uuid];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230425-100703-453867ac-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)